.github/workflows/update-deps.yml

```
@@ -41,10 +41,10 @@ jobs:
           if: env.COMMIT_MADE == 'true'
           uses: cfengine/create-pull-request@v6
           with:
-            title: Updated dependencies
+            title: Updated dependencies for 3.24.x
```

Might as well follow the required / recommended way according to CONTRIBUTING.md;

https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#pull-request-title-and-description

```suggestion
            title: Updated dependencies (3.24)
```